packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4568,13 +4568,21 @@ abstract class ComponentElement extends Element {
   }
 }
 
+T unsafeCast<T>(Object any) => any; // ignore: return_of_invalid_type
```

Comment:
Is this analyzed with implicit downcasts disabled in the analyzer, hence the ignore?  Note that:
1) In dart2js this implicit cast will be ignored
2) In native, this implicit cast will generate identical code to the explicit cast (as T).
In NNBD, you can use `any as dynamic` to get the "implicit cast" behavior in all compilers.